### PR TITLE
vscode dark+ terminal theme 

### PR DESCRIPTION
**Pull request by Senpai-10**
vscode dark+ theme for alacritty

### CODE REVIEW

**Comment by JuanVqz** [comment]
Hi @semantic-release-bot! is this the official VS Code (dark)  color?

**Comment by rajasegar** [comment]
@JuanVqz I think your question is supposed to be for @Senpai-10

**Comment by JuanVqz** [comment]
Ye, it is. Sorry @Senpai-10 😅

**Comment by Senpai-10** [comment]
Hi @JuanVqz 
it has the same colors in vscode dark+ terminal theme

File: themes/vscode.dark.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+colors:
+  name: vscode (dark)
+  author: senpai-10
```
[the remark this answers is not in the document]
Comment (rajasegar) [reply themes/vscode.dark.yml:3]: I am not sure, I think alacritty ignores these values, it only cares about the color values I guess

**Comment by JuanVqz** [comment]
@Senpai-10 The color scheme looks great!
Thank you for your PR! 🚀

**Pull request merged**